### PR TITLE
HW1

### PR DESCRIPTION
**Pull request by ulasonat**
Creating the PR for the part 3 of the HW1.

HW1 Part 1 & 2 has a different PR. Let me know if this was not the preferred way!

### CODE REVIEW

File: numpy.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Open Source Development
```
Comment (timkpaine) [inline numpy.md:1]: move to `projects/python/numpy.md`

Comment (timkpaine) [inline numpy.md:1]: move to `projects/python/numpy.md`

Comment (timkpaine) [inline numpy.md:1]: move to `projects/python/numpy.md`

File: extension.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Open Source Development
```
Comment (timkpaine) [inline extension.md:1]: move to `projects/javascript/prime-video-translate.md` or whatever you'd like to name the library

File: tkp2108.md [hunk 3]
```diff
@@ -4,11 +4,13 @@ Name: Tim Paine
 
 UNI: tkp2108
 
-Github: [link](https://github.com/timkpaine)
+GitHub: [link](https://github.com/timkpaine)
```
Comment (timkpaine) [inline tkp2108.md:7]: undo changes

File: ua2182.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+UNI: ua2182
+
+Github: [link](https://github.com/ulasonat)
+
+GitHub README: [link](https://github.com/ulasonat/ulasonat/blob/main/README.md)
+
+
+## Project Proposal: Extension for Prime Video
+- My goal is to create an extension on Google Chrome for Prime Video that will translate a word from one language to another when clicked on it. There's an extension that does this for Netflix, but Prime Video users could also benefit from that.
+
```
Comment (timkpaine) [inline ua2182.md:14]: add link to project, e.g. `./projects/javascript/prime-video-translater.md`

File: ua2182.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+## Project Contributions
+- Numpy [link](https://github.com/numpy/numpy)
+- First issue to start with: https://github.com/numpy/numpy/issues/22536
```
Comment (timkpaine) [inline ua2182.md:17]: add link to `projects/python/numpy.md`

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Comment by ulasonat** [comment]
I had two pull requests for HW1, which now I see was not necessary. I closed the other pull request which was in main, this pull request (branch hw1_p3) now reflects to all part 1, part 2, and part 3 of the hw1. Let me know if you have questions!

**Pull request closed without merging**